plugin/sign/TODO

```diff
@@ -0,0 +1,4 @@
+* setup function
+* glue, delegation, nsec
+* go-routine, per zone to sign, stop and reload etc
+* more tests
```

Comment:
Auto zone discover for sign similar to auto picking up zones?

Reply:
No, I'm not going to do that here. This just signs